ergotree-ir/src/serialization/expr.rs

```diff
@@ -214,6 +220,9 @@ impl SigmaSerializable for Expr {
                 SigmaPropBytes::OP_CODE => Ok(SigmaPropBytes::sigma_parse(r)?.into()),
                 Tuple::OP_CODE => Ok(Tuple::sigma_parse(r)?.into()),
                 DecodePoint::OP_CODE => Ok(DecodePoint::sigma_parse(r)?.into()),
+                OpCode::BYTE_ARRAY_TO_LONG => Ok(ByteArrayToLong::sigma_parse(r)?.into()),
```

Comment:
I've used `OpCode::XXX` format in the beginning, but later I found `Node::OP_CODE` format more readable. Plus, the latter format "glues" IR node with its opcode.